server/events/terraform/terraform_client.go

```diff
@@ -442,6 +440,7 @@ func (c *DefaultClient) RunCommandAsync(ctx models.ProjectCommandContext, path s
 
 		// Asynchronously copy from stdout/err to outCh.
 		go func() {
+			c.projectCmdOutputHandler.Send(ctx, fmt.Sprintf("\n----- running terraform %s -----\n", args[0]))
```

Comment:
is this for debugging purposes?

Reply:
No, I added for the UI. To see when a new terraform command has started. Looks like this
![Uploading Screen Shot 2021-09-13 at 4.36.35 PM.png…]()

Reply:
and without it, it looks like this 
<img width="1018" alt="Screen Shot 2021-09-13 at 5 14 15 PM" src="https://user-images.githubusercontent.com/43311/133173226-2375c165-8e24-4f0f-9592-ad89a783add1.png">